### PR TITLE
docs: add node cordon information

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: reference/node-corndon.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Node Cordon
```
Comment (avishnu) [inline reference/node-corndon.md]: Typo in the file name.
Should we be specific that this is Mayastor Node Cordon everywhere? Reason is that there is a node cordon functionality in K8s as well.

**Review by chriswldenyer — APPROVED** [review]
LGTM

File: reference/node-corndon.md (hunk 1 above)
Comment (anupriya0703) [reply reference/node-corndon.md]: Makes sense. Will be updating that text.

Comment (anupriya0703) [reply reference/node-corndon.md]: Updated the file name.

**Review by avishnu — APPROVED** [review]
Lgtm

**Pull request merged**